vllm/entrypoints/launcher.py

```diff
@@ -58,6 +62,37 @@ async def dummy_shutdown() -> None:
         return server.shutdown()
 
 
+async def serve_zmq(arg, zmq_server_port: int, app: FastAPI) -> None:
+    """Server routine"""
+    logger.info("zmq Server start arg: %s, zmq_server_port: %d", arg,
+                zmq_server_port)
+    # different zmq context can't communicate use inproc
+    workers_addr = "ipc://workers"
+    clients_addr = f"ipc://127.0.0.1:{zmq_server_port}"
+    # Prepare our context and sockets
+    context = zmq.asyncio.Context.instance()
+    try:
+        tasks = [
+            asyncio.create_task(worker_routine(workers_addr, app, context, i))
```

Comment:
- Why do we need to have 100 workers?
- Since each worker is a coroutine anyways, couldn't we just have one worker that dispatches work?